### ISSUE TITLE
[iOS NetNewsWire] Missing top image on Reader View most of the time

### ISSUE THREAD

**Issue by fede777** [feedbin/feedbin#719]
Hi, I posted this on the NNW GitHub, but was redirected here.

Almost every time I use Reader View on NNW the top image is missing, see below.

![WhatsApp Image 2024-02-08 at 16 12 05](https://github.com/Ranchero-Software/NetNewsWire/assets/7155112/ee5e2ba3-2583-4ced-a6f4-be7280d46126)

Is this a known issue?

**Comment by fede777** [comment]
Anyone?

**Comment by michaelnordmeyer** [comment]
Yes, I see this as well and for quite a long time, probably years. And it is not limited to The Verge.
